generator/internal/discovery_to_proto.cc

```diff
@@ -104,7 +105,12 @@ StatusOr<std::map<std::string, DiscoveryTypeVertex>> ExtractTypesFromSchema(
       schemas_all_type_object = false;
       continue;
     }
-    types.emplace(id, DiscoveryTypeVertex{id, s});
+    types.emplace(id, DiscoveryTypeVertex{
+                          id,
+                          absl::StrFormat("google.cloud.cpp.%s.%s",
```

Comment:
I assume this magic string is well-known, if I should know why it is here just send me the doc link, but maybe add a link for "future us".